doc/src/find_and_immediate.md

```diff
@@ -166,14 +166,15 @@ MIOpen provides a set of Find modes which are used to accelerate the Find calls.
 
 - `NORMAL`, or `1`: Normal Find: This is the full Find mode call, which will benchmark all the solvers and return a list.
 - `FAST`, or `2`: Fast Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, utilize the Immediate mode fallback. If Start-up times are expected to be faster, but worse GPU performance.
-- `HYBRID`, or `3`, or unset `MIOPEN_FIND_MODE`: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times than Fast Find, but no GPU performance drop.
+- `HYBRID`, or `3`: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times than Fast Find, but no GPU performance drop.
+- `OPTIMIZED_HYBRID`, or `4`, or unset `MIOPEN_FIND_MODE`: Optimized hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery with skipping slow-compiling kernels. Faster start-up times than Hybrid Find, may have GPU performance drop.
 
- As of MIOpen 2.6, the default mode is set to `HYBRID` mode as default. To run the full `NORMAL` Find mode, set the environment as:
+ As of MIOpen 2.7, the default mode is set to `OPTIMIZED_HYBRID` mode as default. To run the full `NORMAL` Find mode, set the environment as:
```

Comment:
I dont think it would be a good idea to make this the default mode as this can "corrupt" the find_db with slower solvers. I think we would need a way to annotate entries in find_db with an attribute to know how likely it is to be correct. This way in `HYBRID` mode it can know whether to run a normal find mode or not when an entry is found.

Reply:
I wasn't planning on making it default any time soon. @zjing14 you might want to modify the documentation and make sure this is not default.

Reply:
@pfultz2 Perhaps this design make thing too complicated internally. It might be better to have this mode as our initial try for an immediate mode fall back.

Reply:
Set the "hybrid" mode back to default.